cassandane/Cassandane/Unit/TestCase.pm

```diff
@@ -523,6 +523,27 @@ sub assert_not_file_test
                   "'$path' unexpectedly passed '$test_type' test");
 }
 
+sub assert_cmp_deeply
+{
+    my ($self, $have, $want, $desc) = @_;
+    $desc ||= "deep comparison matched";
+
+    require Test::Deep;
```

Comment:
This effectively make Test::Deep a prereq of Cass, but I think this is okay.  We have basically agreed that JMAP::Tester is a prereq, and it requires it.  So, I think nothing needs changing, just calling it out.